### ISSUE TITLE
EventPipe format compressed event

### ISSUE THREAD

**Issue by iskiselev** [microsoft/perfview#967]
Documentation (src/TraceEvent/EventPipe/EventPipeFormat.md) states that:
```
SequenceNumber optional varint32
if Flags & 2 the value is read from the stream + previous SequenceNumber
otherwise previous sequence number
in either case, if MetadataId != 0 increment by 1
```

Looks like the last line should say SequenceNumber, not MetadataId.
So it should be something like:  
```
sequenceNumber =  previous.SequenceNumber + (flag ? readVarInt32() : 0)
if (sequenceNumber !=0) 
  sequenceNumber++
```

Is it correct interpretation?

**Comment by iskiselev** [comment]
Found this in source. My interpretation was incorrect:
```
                if((flags & (byte)CompressedHeaderFlags.MetadataId) != 0)
                {
                    header.MetaDataId = (int)ReadVarUInt32(ref headerPtr);
                }
                if ((flags & (byte)CompressedHeaderFlags.CaptureThreadAndSequence) != 0)
                {
                    header.SequenceNumber += (int)ReadVarUInt32(ref headerPtr) + 1;
                    header.CaptureThreadId = (long)ReadVarUInt64(ref headerPtr);
                    header.CaptureProcNumber = (int)ReadVarUInt32(ref headerPtr);
                }
                else
                {
                    if(header.MetaDataId != 0)
                    {
                        header.SequenceNumber++;
                    }
                }
```

would it be better to reword last line to something like: "in either case, if MetadataId != 0 increment SequenceNumber by 1"

**Comment by brianrob** [comment]
@noahfalk

**Comment by noahfalk** [comment]
I'm happy to take doc suggestions : ) I created a PR if you could merge it @brianrob?

**Comment by brianrob** [comment]
Fixed via #975.
